pom.xml

```diff
@@ -392,10 +392,10 @@
                         <configuration>
                             <rules>
                                 <requireMavenVersion>
-                                    <version>3.0.5</version>
+                                    <version>[3.5,4.0)</version>
                                 </requireMavenVersion>
                                 <requireJavaVersion>
-                                    <version>1.8.0</version>
+                                    <version>[1.8,1.9)</version>
```

Comment:
It's a `>=` constraint by default—a range makes sense for Maven version just in case of breaking changes, but for Java IMO it would be good to build 1.8 and JDK 11 in CI so we might need something fancier here.

Reply:
Currently I limit it to Java 8 because we're using Beam SDK and I believe it's not fully compatible with newer major versions of Java.

https://issues.apache.org/jira/browse/BEAM-2530

We can support more versions once we're sure it's compatible